libp2p/Cargo.toml

```diff
@@ -107,7 +107,7 @@ libp2p-identity = { version = "0.1.0", path = "../identity" }
 libp2p-kad = { version = "0.43.0", path = "../protocols/kad", optional = true }
 libp2p-metrics = { version = "0.12.0", path = "../misc/metrics", optional = true }
 libp2p-mplex = { version = "0.39.0", path = "../muxers/mplex", optional = true }
-libp2p-noise = { version = "0.42.0", path = "../transports/noise", optional = true }
+libp2p-noise = { version = "0.42.2", path = "../transports/noise", optional = true }
```

Comment:
I am assuming that you are not bumping the `libp2p` version in this pull request as this change does not warrant a new release, i.e. it doesn't need to be included in the next round of patch releases. Is that correct?

Reply:
Yep, that was my thinking. Given that it is backwards compatible, we don't need to cut a release of `libp2p` just because we bumped some dependencies. However, the change in the manifest is important to make sure that once we make a release, we specify the new correct minimum version.